addons/mail/static/src/models/locale/locale.js

```diff
@@ -23,6 +31,12 @@ function factory(dependencies) {
     }
 
     Locale.fields = {
+        /**
+         * Language used by interface, formatted like {language ISO 2}_{country ISO 2} (eg: fr_FR).
+         */
+        language: attr({
+            compute: '_computeLanguage',
+        }),
```

Comment:
This value should be updated in `set_bundle` override, similar to what we do for `textDirection`. (initial value might be wrong, and we can't express dependency here)

Reply:
I'll push the change so we can merge